src/main/resources/org/jenkinsci/plugins/workflow/steps/PushdStep/help-path.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The relative path of the directory in the workspace to use as a new working directory.
+</div>
```

Comment:
It's a good thing to have a newline at the end of any files. This can be done by your code editor automatically so you don't have to remember about it. 

This is not mandatory for this PR to be valid. 

```suggestion
</div>

```